print-battleground-state-changes

```diff
@@ -213,14 +216,28 @@ def html_write_state_head(state: str, state_slug: str, summary: IterationSummary
     '''
 
 def html_summary(state_slug: str, summary: IterationSummary):
+    if summary.counties_partition:
+        counties_partition = sorted(summary.counties_partition.items(), key=lambda x: x[1], reverse=True)
```

Comment:
```suggestion
        counties_partition = sorted(summary.counties_partition.values(), reverse=True)
```

Reply:
If we sorted the values, we'd lose the corresponding county name